examples/app.py

```diff
@@ -62,7 +62,7 @@
         cd static
         npm install
         cd ..
-        npm install -g node-sass clean-css requirejs uglify-js
+        npm install -g node-sass@3.8.0 clean-css@3.4.24 requirejs uglify-js
```

Comment:
When I tried several weeks ago, using `clean-css-cli` wasn't enough... but I can retry with the latest `invenio-assets` and stuff.